bzt/cli.py

```diff
@@ -293,10 +293,21 @@ def __apply_single_override(self, dest, name, value):
         self.log.debug("Applying: [%s]=%s", parts[-1], value)
         if isinstance(parts[-1], string_types) and parts[-1][0] == '^':
             item = parts[-1][1:]
-            if item in pointer:
-                del pointer[item]
+
+            if isinstance(pointer, list):
+                item = int(item)
+                if item < len(pointer):
```

Comment:
I can imagine a scenario where user wants to remove last (or next to last) execution element, and negative indexes will come in handy for that.